ledger-core/v2/application/testwalletapi/handlers.go

```diff
@@ -175,7 +175,43 @@ func (s *TestWalletServer) GetBalance(w http.ResponseWriter, req *http.Request)
 		TraceID: traceID,
 		Error:   "",
 	}
+
 	defer func() { s.mustWriteResult(w, result) }()
```

Comment:
Don't we want to log error if it occurs?
Something like:
`
defer func() { 
if len(  result.Error) != 0{
log( result.Error )
}
s.mustWriteResult(w, result) 
}()
`

Reply:
the error will go to functest, if it is not expected, it will be logged/checked there. In Border contracts we will log contract output for sure

Reply:
Why it's not expected to be logged here?
If there is any problem, I just grep traceID in log and see all path of request